kafka_consumer/tests/docker/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   zookeeper:
-    image: zookeeper:3.6.0
+    image: wurstmeister/zookeeper
```

Comment:
is there a tag we can attach to?

Reply:
@hithwen There is only two choices `3.4.6` from 4 years ago or `latest`: https://hub.docker.com/r/wurstmeister/zookeeper/tags

I think it's fine to have `latest` for this case, `zookeeper` is not the main service we are testing, only a dependency. The change of an upgrade of the image that break this test should be low.